src/components/missions/MissionsList.js

```diff
@@ -0,0 +1,7 @@
+const MissionsList = () => (
```

Comment:
Optional:
-[] I suggest working with the useEffect() hook somewhere in your code, so you can meet the requirements for testing fetching the data. This should be rendered only when the page loads, when the user navigate throughout the links it should not be called again.